mix.exs

```diff
@@ -32,11 +32,10 @@ defmodule Elixometer.Mixfile do
 
   defp deps do
     [
-        {:meck, "~> 0.8.3"},
+        {:meck, "~> 0.8.3", only: :test},
         {:edown, github: "uwiger/edown", tag: "0.7", override: true},
         {:lager, github: "basho/lager", tag: "2.1.0", override: true},
-        {:exometer, github: "pspdfkit-labs/exometer"},
-        {:netlink, github: "Feuerlabs/netlink", ref: "d6e7188e", override: true},
+        {:exometer_core, git: "git://github.com/PSPDFKit-labs/exometer_core.git", branch: "master"},
```

Comment:
As I understand you want to pull from an "offcial" source and avoid merging. 

At some point, you may need to fork and pull from your own and submitt PR back upstream. Lots of those projects are using rebar2 (various versions of rebar2) and now lots are migrating to rebar3. 

Some are linking to master etc. Lots of erlang shops forked their own internally.